torch/ao/nn/quantized/modules/batchnorm.py

```diff
@@ -48,6 +48,47 @@ def from_reference(cls, bn, output_scale, output_zero_point):
         return qbn
 
 
+class BatchNorm1d(_BatchNorm):
+    r"""This is the quantized version of :class:`~torch.nn.BatchNorm1d`."""
+
+    _NNI_BN_RELU_MODULE = nni.BNReLU2d
```

Comment:
BNReLU1d?

Reply:
Thanks for spotting that. It's fixed now.